### PR TITLE
Fix intermittent test failure in aecore_sync_SUITE

### PR DESCRIPTION
**Pull request by hanssv**
Make sure to generate non-overlapping random ports. Also fix cleanup when tests fail (we sometimes have to start nodes to clean up properly).

Also fix some whitespace.

This PR is supported by the Æternity Foundation

### CODE REVIEW

**Review by uwiger — APPROVED** [review]
LGTM

File: apps/aecore/test/aecore_sync_SUITE.erl [hunk 1]
```diff
@@ -1245,9 +1244,23 @@ encode_peer_for_config(Peer) ->
...
 
-random_peer() ->
+%% Make sure to generate non-overlapping ports...
+random_peers(N) ->
+    Ports = random_ports(N, []),
+    [ random_peer(Port) || Port <- Ports ].
+
+random_ports(0, Ports) -> Ports;
+random_ports(N, Ports) ->
+  Port = 4000 + rand:uniform(999),
```
Comment (dincho) [inline apps/aecore/test/aecore_sync_SUITE.erl:1254]: I'd go even further up to ~60000 for a pinch of more randomness

**Review by dincho — APPROVED** [review]
🤞🏻

File: apps/aecore/test/aecore_sync_SUITE.erl [hunk 2]
```diff
@@ -1245,9 +1244,23 @@ encode_peer_for_config(Peer) ->
     Port = integer_to_binary(aec_peer:port(Peer)),
     <<"aenode://", PK/binary, "@", Host/binary, ":", Port/binary>>.
 
-random_peer() ->
+%% Make sure to generate non-overlapping ports...
+random_peers(N) ->
```
[the remark this answers is not in the document]
Comment (hanssv) [reply apps/aecore/test/aecore_sync_SUITE.erl:1248]: You are correct, but each test case is separate as far as I could tell so it is enough if each invocation is unique

I tried to run this with `rand:uniform(4)` and it worked!

File: apps/aecore/test/aecore_sync_SUITE.erl (hunk 1 above)
Comment (hanssv) [reply apps/aecore/test/aecore_sync_SUITE.erl:1254]: Probably not neecessary, I don't even think the randomness brings _anything_ to the table. We could just as well have used 4001-4004 sequentially I think.

**Pull request merged**